client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -33,6 +33,43 @@ const PatternAssembler: Step = ( { navigation } ) => {
 	const siteId = useSiteIdParam();
 	const siteSlugOrId = siteSlug ? siteSlug : siteId;
 
+	const getPatterns = () =>
+		[ header, ...sections, footer ].filter( ( pattern ) => pattern ) as Pattern[];
+
+	const trackEventPatternAdd = ( patternType: string ) => {
+		recordTracksEvent( 'calypso_signup_bcpa_add_pattern_click', {
```

Comment:
Let's do 'calypso_signup_bcpa_pattern_add_click'

To keep with increasing specificity